src/pymorize/units.py

```diff
@@ -29,17 +24,12 @@
 import xarray as xr
 from chemicals import periodic_table
 
-from .frequency import CMIP_FREQUENCIES
 from .logging import logger
 from .rule import Rule
 
 ureg = pint_xarray.unit_registry
 
-
-def assign_frequency_to_unit_registry():
-    """Assign the CMIP6 frequencies to the unit registry."""
-    for freq_name, days in CMIP_FREQUENCIES.items():
-        ureg.define(f"{freq_name} = {days} * d")
+ureg.define("practical_salinity_unit = g/kg = psu = PSU")
```

Comment:
```suggestion
```

Talking to @pgierz we decided to not support `psu` in the end because in models different than FESOM there might be different species of salt and the g/kg equivalence will only be approximate. In that case, the users will need to create a rule themselves to integrate their species of salt and provide and adequate conversion.

Apologies for changing the mind about it @siligam, I have not considered these details when I suggested to support psu.